test/DependencyInjection/Extension/SerializerExtensionTest.php

```diff
@@ -19,7 +19,7 @@ class SerializerExtensionTest extends AbstractExtensionTestCase
     /**
      * {@inheritdoc}
      */
-    protected function getContainerExtensions()
+    protected function getContainerExtensions() :array
```

Comment:
The typehint style seems inconsistent; sometimes you have a space before the hint, sometimes you don't. And I'd like to note that probably the typehints shoudn't be added right now, since the whole bundle is not using those.

Reply:
I think that is necessary for new phpunit versions, in this case we can add this typehint or not?

Reply:
No, AFAIK requiring return type hints is only a good PHP 7.x practice that does not relate to PHPUnit requiring those or anything.

Reply:
You are Right, is not PHPUnit but Symfony dependency injection test library that has added type hint to many functions like getContainerExtensions as you can see here for example:

https://github.com/SymfonyTest/SymfonyDependencyInjectionTest/blob/master/PhpUnit/AbstractExtensionTestCase.php

I had to add all type hint for this reason

Reply:
Uh, right; thanks for taking your time and explaining this. Yup, that's 100% valid then, just need to figure out the correct style (is that in the [coding standard](https://github.com/broadway/coding-standard)?) :)

Reply:
If it is not in the coding standard we can discuss it if it is necessary and if this can be an improvement for this library.
No no, thanks to you to have figured out this possible problem with type hint. It's important to understand why some decision has been taken and discuss it